src/mdns/mdns_avahi.cpp

```diff
@@ -367,9 +367,20 @@ void PublisherAvahi::HandleGroupState(AvahiEntryGroup *aGroup, AvahiEntryGroupSt
         break;
 
     case AVAHI_ENTRY_GROUP_COLLISION:
+    {
         otbrLogErr("Name collision!");
-        CallHostOrServiceCallback(aGroup, OTBR_ERROR_DUPLICATED);
+        Services::iterator serviceIt = FindService(aGroup);
```

Comment:
Can it be a Host name conflict?

Reply:
I think host name conflict should trigger a `AVAHI_SERVER_COLLISION`.